scality/metalk8s
# Node Management: Salt API integration

Scope:
- Salt api login management
- Call SALT API to deploy the first step "deploy salt minion"

How to test:
- Build and Launch the UI 
- Login
- Create a new node via the form
- once created, the node is added to the list. 
- click on the "Deploy" button of the node
- a popup should display the result of the deploy
